nautobot/utilities/utils.py

```diff
@@ -1013,9 +1013,11 @@ def convert_querydict_to_factory_formset_acceptable_querydict(request_querydict,
 
 
 def is_single_choice_field(filterset, field_name):
+    from nautobot.extras.filters import CustomFieldMultiSelectFilter  # Avoid circular imports
+
     # Some filter parameters do not accept multiple values, e.g DateTime, Boolean, Int fields and the q field, etc.
     field = get_filterset_field(filterset, field_name)
-    return not isinstance(field, django_filters.MultipleChoiceFilter)
+    return not isinstance(field, (django_filters.MultipleChoiceFilter, CustomFieldMultiSelectFilter))
```

Comment:
Should we instead change it so that `CustomFieldMultiSelectFilter` is a `MultipleChoiceFilter` subclass?